influxdb_connector/requirements.txt

```diff
@@ -0,0 +1,3 @@
+cnvrgv2
```

Comment:
Once you have pulled the appropriate image, you can add package versions by running the 'pip list' command in your local environment. Please try this on the UI before pushing to the repo.

Reply:
version number is now specified,